webpack.config.js

```diff
@@ -52,7 +49,7 @@ module.exports = async (env, options) => {
           use: "html-loader",
         },
         {
-          test: /\.(png|jpg|jpeg|ttf|woff|woff2|gif|ico)$/,
+          test: /\.(png|jpg|jpeg|gif|ico)$/,
```

Comment:
David found that in some scenarios ttf, woff, and woff2 need to be included.

```suggestion
          test: /\.(png|jpg|jpeg|ttf|woff|woff2|gif|ico)$/,
```

Reply:
Is that specific to react?  They aren't in the other template repos.  The more we can maintain uniformity between them the easier it is to maintain.

Reply:
> Is that specific to react? They aren't in the other template repos. The more we can maintain uniformity between them the easier it is to maintain.

Specific to React. It's a Fluent UI issue: [[node module error when creating outlook web add-in](https://stackoverflow.com/questions/78072626/node-module-error-when-creating-outlook-web-add-in)](https://stackoverflow.com/questions/78072626/node-module-error-when-creating-outlook-web-add-in)

Reply:
I see . . . we'll keep it here, but I suggest making the same change to the taskpane template (and we'll merge it across) so we are consistent and that become one less problem developers hit if they move to use the newer fluent libraries.